### ISSUE TITLE
go get errors

### ISSUE THREAD

**Issue by wkhere** [gijit/gi#25]
Hi!
Just discovered this project and it's a very neat idea.

Still, problems with installation, starting with `go get`:

```
% go get -t -v -u github.com/gijit/gi/...                                                              2
warning: ignoring symlink /home/wk/go/src/github.com/gijit/gi/cmd/gi/prelude
github.com/gijit/gi (download)
github.com/gijit/gi/vendor/github.com/glycerine/golua/lua
github.com/gijit/gi/pkg/compiler/shadow/gonum.org/v1/gonum/blas
github.com/gijit/gi/pkg/compiler/shadow/gonum.org/v1/gonum/diff/fd
github.com/gijit/gi/pkg/compiler/shadow/gonum.org/v1/gonum/floats
# github.com/gijit/gi/pkg/compiler/shadow/gonum.org/v1/gonum/blas
pkg/compiler/shadow/gonum.org/v1/gonum/blas/blas.genimp.go:1:21: syntax error: unexpected ., expecting ;
github.com/gijit/gi/pkg/compiler/shadow/gonum.org/v1/gonum/graph
# github.com/gijit/gi/pkg/compiler/shadow/gonum.org/v1/gonum/diff/fd
pkg/compiler/shadow/gonum.org/v1/gonum/diff/fd/fd.genimp.go:1:21: syntax error: unexpected ., expecting ;
github.com/gijit/gi/pkg/compiler/shadow/gonum.org/v1/gonum/integrate
go build github.com/gijit/gi/vendor/github.com/glycerine/golua/lua: invalid flag in #cgo LDFLAGS: /home/wk/go/src/github.com/gijit/gi/vendor/github.com/glycerine/golua/lua/../../../LuaJIT/LuaJIT/src/libluajit.a
github.com/gijit/gi/pkg/compiler/shadow/gonum.org/v1/gonum/lapack
# github.com/gijit/gi/pkg/compiler/shadow/gonum.org/v1/gonum/floats
pkg/compiler/shadow/gonum.org/v1/gonum/floats/floats.genimp.go:1:21: syntax error: unexpected ., expecting ;
github.com/gijit/gi/pkg/compiler/shadow/gonum.org/v1/gonum/mat
# github.com/gijit/gi/pkg/compiler/shadow/gonum.org/v1/gonum/lapack
pkg/compiler/shadow/gonum.org/v1/gonum/lapack/lapack.genimp.go:1:21: syntax error: unexpected ., expecting ;
github.com/gijit/gi/pkg/compiler/shadow/gonum.org/v1/gonum/optimize
# github.com/gijit/gi/pkg/compiler/shadow/gonum.org/v1/gonum/graph
pkg/compiler/shadow/gonum.org/v1/gonum/graph/graph.genimp.go:1:21: syntax error: unexpected ., expecting ;
# github.com/gijit/gi/pkg/compiler/shadow/gonum.org/v1/gonum/integrate
pkg/compiler/shadow/gonum.org/v1/gonum/integrate/integrate.genimp.go:1:21: syntax error: unexpected ., expecting ;
github.com/gijit/gi/pkg/compiler/shadow/gonum.org/v1/gonum/stat
github.com/gijit/gi/pkg/compiler/shadow/gonum.org/v1/gonum/unit
# github.com/gijit/gi/pkg/compiler/shadow/gonum.org/v1/gonum/mat
pkg/compiler/shadow/gonum.org/v1/gonum/mat/mat.genimp.go:1:21: syntax error: unexpected ., expecting ;
github.com/gijit/gi/pkg/compiler/templates
# github.com/gijit/gi/pkg/compiler/shadow/gonum.org/v1/gonum/stat
pkg/compiler/shadow/gonum.org/v1/gonum/stat/stat.genimp.go:1:21: syntax error: unexpected ., expecting ;
github.com/gijit/gi/pkg/muse
# github.com/gijit/gi/pkg/compiler/shadow/gonum.org/v1/gonum/unit
pkg/compiler/shadow/gonum.org/v1/gonum/unit/unit.genimp.go:1:21: syntax error: unexpected ., expecting ;
# github.com/gijit/gi/pkg/compiler/shadow/gonum.org/v1/gonum/optimize
pkg/compiler/shadow/gonum.org/v1/gonum/optimize/optimize.genimp.go:1:21: syntax error: unexpected ., expecting ;
# github.com/gijit/gi/pkg/compiler/templates
pkg/compiler/templates/flow10.go:19:6: main redeclared in this block
	previous declaration at pkg/compiler/templates/flow1.go:7:6
pkg/compiler/templates/flow11.go:9:6: main redeclared in this block
	previous declaration at pkg/compiler/templates/flow10.go:19:6
pkg/compiler/templates/flow2.go:8:6: main redeclared in this block
	previous declaration at pkg/compiler/templates/flow11.go:9:6
pkg/compiler/templates/flow3.go:8:6: main redeclared in this block
	previous declaration at pkg/compiler/templates/flow2.go:8:6
pkg/compiler/templates/flow4.go:5:6: main redeclared in this block
	previous declaration at pkg/compiler/templates/flow3.go:8:6
pkg/compiler/templates/flow5.go:5:6: main redeclared in this block
	previous declaration at pkg/compiler/templates/flow4.go:5:6
pkg/compiler/templates/flow6.go:5:6: main redeclared in this block
	previous declaration at pkg/compiler/templates/flow5.go:5:6
pkg/compiler/templates/flow7.go:8:5: global redeclared in this block
	previous declaration at pkg/compiler/templates/flow10.go:8:5
pkg/compiler/templates/flow7.go:10:15: deeper redeclared in this block
	previous declaration at pkg/compiler/templates/flow10.go:10:15
pkg/compiler/templates/flow7.go:15:6: main redeclared in this block
	previous declaration at pkg/compiler/templates/flow6.go:5:6
pkg/compiler/templates/flow7.go:15:6: too many errors
# github.com/gijit/gi/pkg/muse
pkg/muse/muse.go:134:3: undefined: pp
pkg/muse/muse.go:146:3: undefined: pp
pkg/muse/muse.go:147:3: undefined: pp
pkg/muse/muse.go:148:3: undefined: pp
pkg/muse/muse.go:149:3: undefined: pp
pkg/muse/muse.go:150:3: undefined: pp
pkg/muse/muse.go:151:3: undefined: pp
pkg/muse/util.go:108:2: undefined: pp
pkg/muse/util.go:109:2: undefined: pp
pkg/muse/util.go:146:2: undefined: pp
pkg/muse/util.go:146:2: too many errors
```

My `go version` is 1.9.4

**Comment by glycerine** [comment]
ah. yes. I was playing with importing gonum and forgot that if you don’t have it installed, you will get such errors. I will add these as explicit dependencies once back at the desk. Then go get will pull them in. For now, you can just install gonum manually.


Jason

**Comment by glycerine** [comment]
This is primarily due to the breakage that go1.9.4 imposes.

**Comment by glycerine** [comment]
I just pushed v0.9.7 which builds for me under go1.9.4. Give it a try and let me know how it goes.

**Comment by wkhere** [comment]
For me it doesn't. What's interesting, also doesn't under Go 1.9.3. The cause seems to be gonum - it fails to compile (I don't have it so far on my boxes). 
Maybe the gonum HEAD is broken right now?

**Comment by glycerine** [comment]
Found a fixed a number of issues. Give tip a try.

**Comment by glycerine** [comment]
Fixed in v0.9.12. Note the new install instructions.
~~~
$ go get -d github.com/gijit/gi/cmd/gi                                                          
$ cd $GOPATH/src/github.com/gijit/gi && make install  
~~~
